guide/glossary.md

```diff
@@ -249,21 +249,39 @@ A standard for multi-signature that uses Schnorr signatures. Previously, the mor
 
 - [Proposal](https://eprint.iacr.org/2018/068) in the Cryptology ePrint archive
 
-### Node
+### Nodes
```

Comment:
there's are some linking opportunities and a fix necessary on the principles page with these additions.

- https://bitcoin.design/guide/designing-products/principles/#decentralization

Also consider most terms in the glossary currently are singular  (besides "Keys") not sure if that was a conscious decision or not, but seems reasonable to think of each object as a singular thing being described vs a group.

Reply:
Fixed the link issue, reverted back to just node instead of nodes. 

Yes it was intentional, I think these are relevant terms that should be grouped together. Keys is singular still (though I was planning on grouping that section also).